api/swagger.yaml

```diff
@@ -7612,6 +7612,15 @@ paths:
                         Nameservers: ["8.8.8.8"]
                         Search: ["example.org"]
                         Options: ["timeout:3"]
+                      Secrets:
+                       -
+                         File:
+                          Name: "www.example.org.key"
+                          UID: "33"
+                          GID: "33"
+                          Mode: 384
```

Comment:
Actually, this was created with `0600` (octal);  which is 384 decimal. The default is `444` (octal), which is `292` decimal